### PR TITLE
feat: add direction selection to sandbox

### PR DESCRIPTION
**Pull request by v4dyar4**

### CODE REVIEW

File: src/components/SandboxComponent/SandboxComponent.tsx [hunk 1]
```diff
@@ -62,13 +62,14 @@ export const SandboxComponent: React.FC<ComponentProps> = ({componentId, libId})
...
     const theme = pageProps.theme || 'dark';
+    const direction = pageProps.direction || 'ltr';
 
     if (!DynamicComponent || !isMounted) {
         return null;
     }
 
     return (
-        <ThemeProvider theme={theme} rootClassName={b('wrapper', null, 'sandbox')}>
+        <ThemeProvider theme={theme} direction={direction} rootClassName={b('wrapper', null, 'sandbox')}>
```
Comment (imsitnikov) [inline src/components/SandboxComponent/SandboxComponent.tsx:72]: Lint error

File: src/content/components/uikit/UserLabel/index.ts [hunk 2]
```diff
@@ -19,7 +20,7 @@ export const userLabelConfig = {
             avatar: {
                 type: 'input',
                 defaultValue:
-                    'https://upload.wikimedia.org/wikipedia/commons/thumb/3/33/Charles_Darwin_by_Julia_Margaret_Cameron%2C_c._1868.jpg/193px-Charles_Darwin_by_Julia_Margaret_Cameron%2C_c._1868.jpg',
+                  'https://upload.wikimedia.org/wikipedia/commons/thumb/3/33/Charles_Darwin_by_Julia_Margaret_Cameron%2C_c._1868.jpg/193px-Charles_Darwin_by_Julia_Margaret_Cameron%2C_c._1868.jpg',
```
Comment (imsitnikov) [inline src/content/components/uikit/UserLabel/index.ts:23]: Lint error

**Review by imsitnikov — APPROVED** [review]

File: src/components/SandboxComponent/SandboxComponent.tsx (hunk 1 above)
Comment (v4dyar4) [reply src/components/SandboxComponent/SandboxComponent.tsx]: I just fixed it =)

File: src/content/components/uikit/UserLabel/index.ts (hunk 2 above)
Comment (v4dyar4) [reply src/content/components/uikit/UserLabel/index.ts]: it was in outdated commit, it is no longer there

**Pull request merged**